css/default.scss

```diff
@@ -90,7 +90,7 @@ main {
 }
 
 section > p {
-  padding: 0 5rem;
+  // padding: 0 5rem;
```

Comment:
This only applied at really small sized thanks to your breakpoints and lead to weird things as 5 rem is really big